include/fuzzuf/utils/parallel_mode.hpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * fuzzuf
+ * Copyright (C) 2021 Ricerca Security
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+/**
+ * @file parallel_mode.hpp
+ * @author Ricerca Security <fuzzuf-dev@ricsec.co.jp>
+ */
+#ifndef FUZZUF_INCLUDE_UTILS_PARALLEL_MODE_HPP
+#define FUZZUF_INCLUDE_UTILS_PARALLEL_MODE_HPP
+
```

Comment:
It's acceptable to place them under utils, but need to move them in the future.
```suggestion

// FIXME: this should be moved to algorithms/afl/?
```